jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/MobileUtils.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.jdi.light.mobile;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.interfaces.base.IClickable;
+import com.epam.jdi.light.elements.interfaces.base.INamed;
+import com.epam.jdi.light.elements.interfaces.common.IsButton;
+import com.epam.jdi.light.mobile.elements.base.MobileAppUIElement;
+import com.epam.jdi.light.mobile.elements.common.app.ios.MenuItem;
+import com.epam.jdi.light.mobile.interfaces.HasTouchActions;
+import com.epam.jdi.tools.func.JFunc2;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.lang.reflect.Field;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import static com.epam.jdi.light.common.Exceptions.exception;
+import static com.epam.jdi.light.driver.WebDriverFactory.getDriver;
+import static com.epam.jdi.light.mobile.elements.init.MobileAppFactory.$;
+import static com.epam.jdi.tools.LinqUtils.*;
+import static com.epam.jdi.tools.ReflectionUtils.*;
+import static com.epam.jdi.tools.StringUtils.namesEqual;
+
+/**
+ * This class introduces the wrappers around the driver methods.
+ * Since each mobile driver has its own set of features, every time some
+ * driver method is called, we need to perform a check if the current driver is able to perform
+ * the desired action. These methods perform such check and fire an exception if the method
+ * cannot be called with the current driver
+ */
+public class MobileUtils {
+    public static <I> void executeDriverMethod(Class<I> clazz, Consumer<I> consumer) {
+        WebDriver driver = getDriver();
+        if (clazz.isInstance(driver)) {
+            consumer.accept(clazz.cast(driver));
+        } else {
+            throw exception("Cannot use this method. The driver needs to extend/implement the " + clazz.getName());
+        }
+    }
+    public static <I, R> R executeDriverMethod(Class<I> clazz, Function<I, R> function) {
+        WebDriver driver = getDriver();
+        if (clazz.isInstance(driver)) {
+            return function.apply(clazz.cast(driver));
+        } else {
+            throw exception("Cannot use this method. The driver needs to extend/implement the " + clazz.getName());
+        }
+    }
+
+
+    public static JFunc2<Object, String, HasTouchActions> GET_DEFAULT_BUTTON =
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6361799)